allofplos/makedb.py

```diff
@@ -20,11 +21,10 @@
 from .transformations import filename_to_doi, convert_country
 from .article_class import Article
 
-
 # TODO: this may need to be updated to take into account the new get_corpus_dir() logic.
 # It is not clear, since this is a relative path from the package how this should work without
-# diving into the code more thoroughly 
-corpusdir = 'allofplos/allofplos_xml'
+# diving into the code more thoroughly
+corpusdir = os.path.join(os.path.dirname(__file__), 'allofplos_xml')
```

Comment:
The corpus dir can be anywhere on the file system, so it should be `corpusdir = get_corpusdir()`